compose/nginx.conf

```diff
@@ -8,6 +8,7 @@ http {
             proxy_pass        http://portal-ui;
             proxy_set_header  Host $http_host;
             proxy_set_header  X-Forwarded-Proto $scheme;
+            proxy_http_version  1.1;
```

Comment:
Can you move this below the comment for `proxy_set_header`, and add another comment? Something like:
```
gzip_http_version sets the minimum HTTP version of a request required to compress a response to 1.1 by default.
proxy_http_version needs to be >= that, but is 1.0 by default.
```